src/app/util/af.h

```diff
@@ -79,6 +79,8 @@
 /** @name Attribute Storage */
 // @{
 
+#define emberEndpointNotFound (0xFFFF)
```

Comment:
please use `kCamelCase` naming for constants. This looks like a function. Probably `kEmberEndpointNotFound` is a good name.

Also prefer `static constexpr EndpointId kEmberEndpointNotFound = 0xFFFF` to macro defines if it compiles since that provides some type safety (this is assuming EndpointId is defined here ... don't want to yak shave too much)